### ISSUE TITLE
 Issue on GAworkshop/contents/03_sexdet/sexdet.rst

### ISSUE THREAD

**Issue by shangshanzhizhe** [stschiff/GAworkshop#3]
Hi Stephan Schiffels, 

I'm referring your coverage calculation method in the sexdet.rst and have a question. The file was referred by the [paper]( https://doi.org/10.1093/molbev/msy220), and I think that you misunderstood the Coverage and Depth.

In your sexDetermination.awk example:
if we use the 'xReads += cov', the xReads is how many times the Reads totally covered at the selected sites, and the 'xReads / xSites' will get the average depth of selected sites. But It means "Covered" when there IS Reads mapped on the site.
So Maybe the 'xReads += cov' should be 'xReads += cov' (**changed to 'xReads += 1'** ) with the judgment 'if cov > 0'.

May I have your suggestion on this issue? Thanks a lot! By the way, your MSMC is excellent~

Shangzhe

**Comment by stschiff** [comment]
Hi Shangzhe. 

I don’t understand. The two lines

xReads += cov

and

	xReads += cov if cov > 0

are exactly the same. If you add 0, it’s as if you don’t add anything, so it’s the same.

Stephan

**Comment by shangshanzhizhe** [comment]
Hi Stephen,

Sorry for the mistake. It should be xReads += 1 if cov > 0.

Shangzhe

**Comment by stschiff** [comment]
No, I think it should be xReads += cov. You want to calculate the mean coverage, and not-covered sites should not contribute. You want to compute

"Total number of reads mapping on sites” divided by “total number of sites considered”

Stephan
